lib/PrefsFeed.jsm

```diff
@@ -84,6 +84,31 @@ this.PrefsFeed = class PrefsFeed {
       value: handoffToAwesomebarPrefValue,
     });
 
+    const discoveryStreamEnabled = Services.prefs.getBoolPref(
+      "browser.newtabpage.activity-stream.discoverystream.enabled"
+    );
```

Comment:
errors when pref not found, need a fall back value for both discoverystream.enabled and discoverystream.hardcoded-basic-layout e.g. Services.prefs.getBoolPref("browser.newtabpage.activity-stream.discoverystream.enabled", false); This still leaves enabling in nigthly and beta when pref not found unresolved. Should we set discoverystream.enabled pref in nightly and beta with value as true?

Reply:
Ah so that's what's wrong.

Reply:
and may be setting[ discoverystream.enabled true here](https://github.com/mozilla/activity-stream/blob/master/lib/ActivityStream.jsm#L140) as DS is on by default in nightly and beta

Reply:
Also I think we're fixing the prefs for nightly/beta and default prefs in the uplift. That said, I remember hearing that it needs to be off in beta for the rollout to work properly.